src/en/articles/how-to-test-android.md

```diff
@@ -0,0 +1,104 @@
+
+A detailed description of these actions is available on the following page: https://a11y-guidelines.orange.com/en/mobile/android/talkback/
+
+By using TalkBack, you can verify that all the essential information for understanding and navigation is rendered by the screen reader, including:
+ - interactive elements (buttons, links, checkboxes, etc.),
+ - the status of the elements (checked or not, selected or not, unfolded or not, etc.),
+ - the page titles must be read to allow users to be informed of a screen change and understand the context,
+ - content changes or temporary messages must also be read (alerts, errors, messages, etc.),
+ - the consistency between VoiceOver reading order and the visual order of information presentation.
+
+
+### Keyboard navigation
+It is possible to use your phone only with an external keyboard.
+1. Connect a keyboard to the phone.
+2. Now, all features available by touch gestures should also be operable with the keyboard.
+Navigating is similar to a desktop keyboard, ```TAB``` key to move forward, ```SHIFT+TAB``` to go back, ```SPACE``` or ```ENTER``` to activate an element. 
+
+Combinations are also specific to the Android mobile environment:
+
+   ![Shortcuts examples: CMD+ENTER for Home, CMD+DEL for Back, ALT+TAB for Recents, ALT+F4 to close the current application, CMD+/ to access keyboard shortcuts.](https://github.com/user-attachments/assets/99bccc16-b3fb-4af8-b8c2-6e712b4367eb)
+
+
+
+3. Keyboard navigation also allows you to check that the navigation order is consistent.
+
+
+### Voice Access
+When "Voice Access" is enabled, you can say commands like:
+- "Go home "
+- "Tap *item name*"
+- "Open *app name*"
+- "Turn volume up"
+
+By default, element names are superimposed.  Also, for an application to be controllable in this way, the interactive components must have a simple and consistent accessible name (in particular in the case of a link image or button image without a visible label).
+
+![](https://github.com/user-attachments/assets/6e28c394-bcdb-4937-9f37-083327e0208a)
+
+
+It's also possible to display the number of the elements or even a numbered grid to reach them.
+
+
+### Font size
+1. Increase the text size from the menu Settings > Display > Font size and style
+
+   ![](https://github.com/user-attachments/assets/759344d4-c227-4401-8dd9-bb75e756aa2d)
+
+
+2. Maximum font size corresponds to a 200% zoom.
+
+3. If the application has been designed to support enlargement (Dynamic Type), then the texts will be correctly rendered, readable, and without loss of information (no truncated text or superposition).
+
+   ![](https://github.com/user-attachments/assets/aaadf571-bb66-4455-9525-bd21df31bacc)  ![](https://github.com/user-attachments/assets/bac1f246-faeb-4cfc-b5df-4084c530d4e2)
+
+
+### Dark mode
+As the dark mode is used more and more, so it's strongly recommended to test your application by activating “dark mode”.
```

Comment:
1. simples quotes only in english so 'dark mode'.
2. As the dark mode is used more and more, it's strongly recommended to test your application by activating 'dark mode' OR As the dark mode is used more and more, it's strongly recommended to test your application by activating 'dark mode'.